app_dart/lib/src/request_handlers/flaky_handler_utils.dart

```diff
@@ -418,11 +418,12 @@ TestOwnership getTestOwnership(String builderName, BuilderType type, String test
 
 /// Gets the [BuilderType] of the builder by looking up the information in the
 /// ci.yaml.
-BuilderType getTypeForBuilder(String? builderName, YamlMap ci) {
-  final List<dynamic>? tags = _getTags(builderName, ci);
+BuilderType getTypeForBuilder(String? builderName, CiYaml ciYaml) {
```

Comment:
builder -> target

Generally, everywhere should refer to everything as target now instead of builder

Reply:
Ah okay. I'll clean it up.

Reply:
Do you mean everything is called target now. The builder terminology is no longer used? For example there are objects like BuilderType, should those be named as TargetType as well?

Reply:
@keyonghan would have a better idea. I'm not sure what `BuilderType` is meant to encapsulate here.

Reply:
BuilderType is an enum and probably fine to leave.